helm/hpcc/templates/eclagent.yaml

```diff
@@ -88,7 +88,7 @@ data:
             workingDir: /var/lib/HPCCSystems
             volumeMounts:
 {{ include "hpcc.addConfigMapVolumeMount" .me | indent 12 }}
```

Comment:
I am wondering if something this sequence should be included by all components, and a similar one for the volumes:

```
{{ include "hpcc.addConfigMapVolumeMount" .me | indent 12 }}
{{ include "hpcc.addVolumeMounts" . | indent 12 }}
{{ include "hpcc.addSecretVolumeMounts" . | indent 12 }}
{{ include "hpcc.addCertificateVolumeMount" (dict "root" .root "name" .me.name "component" $apptype) | indent 12 }}
```

?

Reply:
quite possibly, maybe next iteration of rationalization.

This PR was primarily supposed to be a bug fix for the "spill" which wasn't being handled properly, but it's creeping..